testdata/tree/with_child_directories/src/methods.rs

```diff
@@ -0,0 +1,69 @@
+//! Demonstrate mutation of impl methods.
+
+#![allow(clippy::blacklisted_name)] // "Foo" is just an example name.
+
+use std::fmt;
+
+struct Foo {
+    i: u32,
+}
+
+impl Foo {
+    pub fn new() -> Foo {
+        Foo { i: 32 }
+    }
+
+    pub fn double(&mut self) {
+        self.i *= 2;
+    }
+}
+
+impl fmt::Display for Foo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Foo {}", self.i)
+    }
+}
+
+impl fmt::Debug for &Foo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "&Foo {}", self.i)
+    }
+}
+
+impl Default for Foo {
```

Comment:
Each function here will generate a mutation (or possibly more than one) and each of them will make the test suite slower if we actually test the mutations, especially on Windows. It won't be terrible but we might as well keep it relatively fast unless there's a reason.

I think this is copied from another tree that was trying to test mutation of impls, so it has several of them. 

If the point of this tree is to test directory structures then it's probably enough to just have one function per module.

Also it'll make the expected output a bit smaller and easier to read.

Reply:
I'll merge this and fix it separately.